docs/uperf.md

```diff
@@ -53,8 +55,12 @@ $ oc adm policy add-scc-to-user privileged -z benchmark-operator
 
 `pin_client` what node to pin the client pod to.
 
+`multus[1]` Configure our pods to use multus.
```

Comment:
Not be nitpicky, but can we add experimental tag or something to this, we are not gonna test it in CI as of now, so would be better if we could probably add that.